catalyst/dl/callbacks/criterion.py

```diff
@@ -186,20 +193,35 @@ def __init__(
         self.loss_keys = loss_keys
 
         self.multiplier = multiplier
-        if loss_aggregate_fn == "sum":
-            self.loss_fn = lambda x: torch.sum(torch.stack(x)) * multiplier
-        elif loss_aggregate_fn == "weighted_sum":
+
+        if loss_keys in ("sum", "mean"):
+            if loss_keys is not None and not isinstance(loss_keys, list):
+                raise ValueError(
+                    "For `sum` or `mean` mode the loss_keys must be "
+                    "None or list or str (not dict)"
+                )
+        elif loss_keys in ("weighted_sum", "weighted_mean"):
             if loss_keys is None or not isinstance(loss_keys, dict):
                 raise ValueError(
-                    "For `weighted_sum` mode the loss_keys must be specified "
+                    "For `weighted_sum` or `weighted_mean` mode "
+                    "the loss_keys must be specified "
                     "and must be a dict"
                 )
+
+        if loss_aggregate_fn in ("sum", "weighted_sum", "weighted_mean"):
             self.loss_fn = lambda x: torch.sum(torch.stack(x)) * multiplier
+            if loss_aggregate_fn == "weighted_mean":
```

Comment:
It will be processed correctly, in all 3 cases the final aggregation function is `torch.sum`, but if `isinstance(loss_keys, dict)` it will be weighted